apis/python/src/tiledbvcf/binding/libtiledbvcf.cc

```diff
@@ -53,6 +53,7 @@ PYBIND11_MODULE(libtiledbvcf, m) {
       .def("get_materialized_attributes", &Reader::get_materialized_attributes)
       .def("get_sample_count", &Reader::get_sample_count)
       .def("get_sample_names", &Reader::get_sample_names)
+      .def("get_stats_results", &Reader::get_stats_results)
```

Comment:
In some places, we refer to "stats" and others we refer to "variant_stats". Let's standardize on "variant_stats" since it's more descriptive.